src/main/cdp/connection.ts

```diff
@@ -139,8 +139,8 @@ export class Connection {
         return target;
     }
 
-    protected attachIfNeeded({ targetId }: CdpTargetInfo) {
-        if (this.targetSessions.has(targetId)) {
+    protected attachIfNeeded({ targetId, type }: CdpTargetInfo) {
+        if (this.targetSessions.has(targetId) || type === 'background_page' || type === 'service_worker') {
```

Comment:
Chrome Manifest V3 extensions have a background ServiceWorker of type 'service_worker', instead of a page of type 'background_page'.